python-venv/cmstools/test/cfs_sessions_rc_test/cfs/session.py

```diff
@@ -0,0 +1,247 @@
+        logger.exception("Request timed out for GET query to %s", url)
+        return SessionGetWithNameContainsResult(status_code=0, timed_out=True, error_message="Request timed out")
+    except Exception as exc:
+        logger.exception("Exception during CFS session get: %s", str(exc))
+        return SessionGetWithNameContainsResult(status_code=0, error_message=str(exc))
+
+
+def get_cfs_sessions_list(cfs_session_name_contains: str, cfs_version: CFS_VERSIONS_STR, limit: int, retry: bool = True) -> SessionGetWithNameContainsResult:
+    """
+    Returns a list of CFS sessions with the specified name prefix and pending status.
+    """
+    if cfs_version == "v2":
+        return get_all_cfs_sessions_v2(
+            cfs_session_name_contains=cfs_session_name_contains,
+            cfs_version=cfs_version, retry=retry)
+
+    return get_all_cfs_sessions_v3(
+        cfs_session_name_contains=cfs_session_name_contains,
+        cfs_version=cfs_version,
+        limit=limit, retry=retry)
+
+
+def delete_cfs_sessions(cfs_session_name_contains: str, cfs_version: CFS_VERSIONS_STR, limit: int) -> None:
+    """
+    Delete all CFS sessions with the specified name prefix and pending status.
+    """
+    result = get_cfs_sessions_list(cfs_session_name_contains, cfs_version, limit)
+    if result.status_code != HTTP_OK or not result.session_data:
+        logger.info("No CFS sessions found with name prefix %s and status pending to delete", cfs_session_name_contains)
+        return
+
+    params = {
+        "status": "pending",
+        "name_contains": cfs_session_name_contains
+    }
+    url = CFS_SESSIONS_URL_TEMPLATE.format(api_version=cfs_version)
+    resp = request("delete", url=url, params=params)
+    if resp.status_code == HTTP_OK:
+        logger.info("Deleted CFS sessions with name prefix %s and status pending", cfs_session_name_contains)
+        return
+
+    if resp.status_code == HTTP_BAD_REQUEST:
+        return
+
+    logger.error("Unexpected return code %d from Delete to %s: %s", resp.status_code, url, resp.text)
+    raise CFSRCException()
+
+
+def delete_cfs_session_by_name(cfs_session_name: str, cfs_version: CFS_VERSIONS_STR) -> None:
+    """
+    Delete a CFS session by name.
+    """
+    url = CFS_SESSIONS_URL_TEMPLATE.format(api_version=cfs_version)
+    cfs_sessions_url = f"{url}/{cfs_session_name}"
+    resp = request("delete", cfs_sessions_url)
+    if resp.status_code == HTTP_NO_CONTENT:
+        logger.info("Deleted CFS session %s", cfs_session_name)
+        return
+
+    if resp.status_code == HTTP_NOT_FOUND:
+        logger.info("CFS session %s not found to delete", cfs_session_name)
+        return
+
+    logger.error("Unexpected return code %d from Delete to %s: %s", resp.status_code, url, resp.text)
+    raise CFSRCException()
+
+
+def create_cfs_session(session_name: str, cfs_version: CFS_VERSIONS_STR,
+                       session_payload: dict, expected_http_status: int = 200) -> dict:
```

Comment:
Is there a reason why in this file you sometimes use `HTTP_OK` and sometimes use `200`?